docroot/themes/humsci/humsci_basic/templates/page.html.twig

```diff
@@ -26,7 +26,8 @@
 </div>
 
 {% if page.footer %}
-  <div id="footer">
+  <footer id="footer">
```

Comment:
ah yes. So I do think you need to switch this back to a div. The other stuff that can go above the footer could be almost anything, alerts, messages, another local footer, etc. I think the `footer` tag should go in your template file and replace `section`

let me just check with Bryan to make sure this logic makes sense though: @bryanbraun

Reply:
This is technically the footer for the page so this change does make sense. Semantically, other sections can also have their own footer, they just have to be in a valid HTML element and not a `div`.

Reply:
The question is, what should go insdie of `<footer>`? The global footer, the local footer, or both?

Examples with each footer:
<img width="1345" alt="Screen Shot 2020-02-14 at 11 23 54 AM" src="https://user-images.githubusercontent.com/1256329/74549291-60820100-4f1d-11ea-8cbe-ea8c7aac1dea.png">

With @silverli 's recommendation, only the global footer goes in `<footer>`. With Kasey's, they both go in there.

I could see us going either way. I think what Lindsay's saying is it's safer to just wrap the global footer with `<footer>`, because it'll always be appropriate. If we wrap both footers with `<footer>`, then lots of stuff could end up in the `<footer>` tag which may not be appropriate (since H&S can put arbitrary content in the local footer). I think I agree with @silverli  here.

I do want to recommend we move our global footer include outside of the local footer conditional so our global footer doesn't disappear if there's nothing in the local footer:
```diff
{% if page.footer %}
   {# other content #}
- {% include humsci_basic ~ '/themes/humsci/humsci_basic/templates/components/global-footer.html.twig' with { modifier_class : global_footer_variant_classname } %}
{% endif %}
+ {% include humsci_basic ~ '/themes/humsci/humsci_basic/templates/components/global-footer.html.twig' with { modifier_class : global_footer_variant_classname } %}
```